src/helics/network/zmq/ZmqComms.cpp

```diff
@@ -111,7 +111,7 @@ namespace zeromq {
         zmq::socket_t pullSocket(ctx->getContext(), ZMQ_PULL);
         pullSocket.setsockopt(ZMQ_LINGER, 200);
         zmq::socket_t controlSocket(ctx->getContext(), ZMQ_PAIR);
-        std::string controlsockString = std::string("inproc://") + name + "_control";
+        std::string controlsockString = std::string("inproc://") + getRandomID() + "_control";
```

Comment:
Would making this a combination of name + random ID to guarantee no collisions if the user chooses different names be excessive?

Reply:
This is a control socket name,  I don't see many issues with adding the name in, but I don't think it is necessary.  It is extremely unlikely a 10 digit random code would be duplicated on the same machine.  But we could add it back in for identification purposes if you think that would be useful.

Reply:
That would be useful if the control sock string appears in any log files.